cfme/tests/ansible/test_embedded_ansible_automate.py

```diff
@@ -419,3 +421,43 @@ def test_variable_pass(request, appliance, setup_ansible_repository, import_data
         provision_request = appliance.collections.requests.instantiate(request_description)
         provision_request.wait_for_request(method="ui")
         request.addfinalizer(provision_request.remove_request)
+
+
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(automates=[1677575])
+@pytest.mark.parametrize(
+    "import_data",
+    [DatastoreImport("bz_1677575.zip", "bz_1677575", None)],
+    ids=["datastore"],
+)
+def test_import_export_domain_with_ansible_method(request, appliance, setup_ansible_repository,
+                                                  import_data):
+    """This test case tests support of Export/Import of Domain with Ansible Method
+
+    Bugzilla:
+        1677575
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseimportance: high
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.11
+        casecomponent: Automate
+        tags: automate
+    """
+    # Download datastore file from FTP server
+    fs = FTPClientWrapper(cfme_data.ftpserver.entities.datastores)
+    file_path = fs.download(import_data.file_name)
+
+    # Import datastore file to appliance
+    datastore = appliance.collections.automate_import_exports.instantiate(
+        import_type="file", file_path=file_path
+    )
+    domain = datastore.import_domain_from(import_data.from_domain, import_data.to_domain)
```

Comment:
**Question:**
Why you're explicitly writing these steps.?
There is fixture to achieve this steps. --> [https://github.com/ManageIQ/integration_tests/blob/master/cfme/fixtures/automate.py#L81-L110](url)

Reply:
Yeah. But I cant use this fixture. If you see [here](https://github.com/ManageIQ/integration_tests/blob/master/cfme/fixtures/automate.py#L106); I am not importing domain successfully, I am checking for error flash message on that page.

Reply:
Okay.